CODEOWNERS

```diff
@@ -370,6 +371,7 @@
 /dts/arm/silabs/efm32pg1b*                @rdmeneze
 /dts/arm/silabs/efr32mg21*                @l-alfred
 /dts/arm/silabs/efr32fg13*                @yonsch
+/dts/arm/silabs/efr32bg22*                @sateeshkotapati
```

Comment:
I don't think, that this is a good idea to add yourself as maintainer without additional approvement from main dev group. @galak?

Reply:
How to get additional approval from main dev group. It gave "check patch error" when I don't add myself here.

Reply:
Please read this doc first to understand responsibilities: https://docs.zephyrproject.org/latest/development_process/project_roles.html 

But I would say, that in your case the root of the problem is somewhere else.

Reply:
I will be maintaining the code for EFR32BG22 in future.